src/lib/steps/assets.ts

```diff
@@ -36,7 +37,7 @@ export const processAssets = (src: string, dest: string): Promise<any> => {
         styleProcessor: (path, ext, file, cb) => {
 
           debug(`render stylesheet ${path}`);
-          const render = pickRenderer(path, ext, file);
+          const render = pickRenderer(path, ext, file, src);
```

Comment:
Thanks for the pull request! Nice to have this feature.

One question: is the `src` location here used as the `ctx` for stylus to resolve relative imports?

I think it's a good idea to have a test for a relative import or at least put a code comment somewhere in this file.

Reply:
Thanks for review.
I'll add a comment and test suite right now.